src/components/ExerciseOneForm.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React  from 'react';
```

Comment:
`{ Proptypes }` was also missing here, I'm not sure if that was a mistake or intentional but I quite liked it as a little "side bug" of the exercise 👍